itests/eth_api_test.go

```diff
@@ -0,0 +1,35 @@
+package itests
+
```

Comment:
Don't have a strong opinion on this but maybe move this to fevm_address_test.go and rename that file to eth_address_test.go since that name does cover this domain

Reply:
I think we need to have a dedicated test suite for the EthAPI, my hope is that this will be the start of that.

Reply:
Fair point but the Eth API seems very large to stick in a single file?